libs/remix-lib/src/execution/eventsDecoder.ts

```diff
@@ -36,29 +38,29 @@ class EventsDecoder {
     this._decodeEvents(tx, receipt.logs, contract, contracts, cb)
   }
 
-  _eventABI (contract) {
-    const eventABI = {}
+  _eventABI (contract): Record<string, unknown> {
```

Comment:
we can have a minimum type here `{ event, inputs, object, abi }` instead of `unknown`

Reply:
ok